### PR TITLE
React capstone project - Metrics Webapp

### PR DESCRIPTION
**Pull request by DJ-MrJay**
In this project, I:
- Select an API that provides numeric data about some game series characters
- Implemented one page with a list of items that could be filtered by a search parameter
- Implemented one page for the item details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
...
+- API
+
+## Live Demo
+Live demo on [netlify]()
+
+## Getting Started
+To get a local copy up and running follow these steps
+- Clone the repository using the GitHub link provided below.
+- In your terminal, run: <code>git clone git@github.com:DJ-MrJay/microverse-capstone-3.git</code>
+- Run tests. Follow these [instructions](https://github.com/microverseinc/linters-config). A GitHub action is also set to run during pull request
```
Comment (Reem-lab) [inline README.md]: - [ ] I think the README file is updated so kindly update it to the provided template from microverse you can check this link and please update all the sections to make it descriptive and professional 🚀✨💯🔥 [Readme](https://github.com/microverseinc/readme-template)

File: src/components/__tests__/App.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    });
+    expect(headingElement).toBeInTheDocument();
+  });
+
+  test('renders textbox', () => {
+    render(<MockApp />);
+    const textBoxElement = screen.getByRole('textbox');
+    expect(textBoxElement).toBeInTheDocument();
+  });
+
```
Comment (Reem-lab) [inline src/components/__tests__/App.test.js:29]: - [ ] Great job working on the test but I think the tests are failed here after running `npm run test` all the tests didn't passed so kindly check the errors and try to solve them to manage test your api and your components properly ✨🚀🔥🔥🔥

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @DJ-MrJay, 👋🏻👋🏻

Good job so far! 💯🔥🚀✨
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No Linters error 🔥🔥
- Great working on the presentation video 🔥
- Great job working on the Additional requirements 🚀💯✨✨
- Great job on the functionality 🔥🔥
- Descriptive PR 🔥
- Great design ✨✨✨🚀🚀🚀



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,61 @@
...
+## Project Presentation
+- [Video link](https://www.loom.com/share/de9c94bfa0bc4c30a837434505cecb23)
+
+## Built With
+- Languages: HTML5, CSS3, JavaScript
+- Front-end library: React & Redux
+- API
+
+## Live Demo
+Live demo on [netlify]()
```
Comment (Reem-lab) [inline README.md]: - [ ] I think you forget to deploy your project so kindly make sure to deploy your project to make it easier for the users to say it online and that's very important in the future to work on the production in your Projects 🙏🏻🚀💯💯

**Review by Reem-lab — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (DJ-MrJay) [reply README.md]: Will deploy after I merge

**Review by DeliceLydia — APPROVED** [review]
## Hi @DJ-MrJaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — COMMENTED** [review]
## Hi @DJ-MrJay, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**